source/DistanceMap.cpp

```diff
@@ -128,44 +127,20 @@ set<const System *> DistanceMap::Systems() const
 
 
 
-// Return the destination system - the 'center' system.
-const System *DistanceMap::End() const
-{
-	return center;
-}
-
-
-
-int DistanceMap::RequiredFuel(const System *system1, const System *system2) const
-{
-	auto it1 = route.find(system1);
-	auto it2 = route.find(system2);
-	if(it1 == route.end() || it2 == route.end())
-		return -1;
-	return abs(it1->second.fuel - it2->second.fuel);
-}
-
-
-
-DistanceMap::Edge::Edge(const System *system)
-	: next(system)
-{
-}
-
-
-
-// Sorting operator to prioritize the "best" edges. The priority queue
-// returns the "largest" item, so this should return true if this item
-// is lower priority than the given item.
-bool DistanceMap::Edge::operator<(const Edge &other) const
+// Get the planned route from center to this system.
+vector<const System *> DistanceMap::Plan(const System &target) const
 {
-	if(fuel != other.fuel)
-		return (fuel > other.fuel);
-
-	if(days != other.days)
-		return (days > other.days);
+	auto plan = vector<const System *>{};
```

Comment:
Won't a vector be empty by default?
```suggestion
	vector<const System *> plan;
```